src/lib/handlebars-converter/handlebars-converter.spec.js

```diff
@@ -17,8 +17,8 @@ describe('HandlebarsConverter', function () {
     var handlerbarExistingInstance = HandlebarsConverter.instance(false, path.join(__dirname, '../../templates'));
 
     var templatesMap = {};
-    templatesMap['Resources/ADT_A01/Patient.hbs'] = '{"key1":"value1"}';
-    templatesMap['Resources/ADT_A01/Encounter.hbs'] = '{"key2":"value2"}';
+    templatesMap['Resources/Patient.hbs'] = '{"key1":"value1"}';
```

Comment:
What happens to the code if they remove the patient resource template for some reason?

Reply:
it's just a test code. Users can update their repo if the related template is removed and they still want to maintain the test success rate in their repo.